### PR TITLE
YIELD App submission

### PR DESCRIPTION
**Pull request by khkwan0**
https://www.yield.app/

### CODE REVIEW

**Comment by xrn** [comment]
Hi @khkwan0 can you use please address in EIP-55 format?

**Comment by khkwan0** [comment]
Hello, I've updated the address to be EIP-55 compliant.

**Review by xrn — APPROVED** [review]

**Comment by khkwan0** [comment]
Hello, is there anything else I need to do to facilitate integration?

**Comment by xrn** [comment]
There is overlap for symbols - we are checking internally how to handle this. I will back to you as soon as we will clarify answer

**Comment by khkwan0** [comment]
Checking in, any resolution?

**Comment by khkwan0** [comment]
Hello, checking in.

**Comment by andreimko** [comment]
@xrn Hello, you have any idea how to push this into merge? MetaMask support totally ignores this issue. They replied once, and now ignoring our messages.

**Comment by andreimko** [comment]
> There is overlap for symbols - we are checking internally how to handle this. I will back to you as soon as we will clarify answer

@xrn Have you managed to solve this matter ?

**Comment by andreimko** [comment]
@randymetamask When this issue will be solved?

**Comment by andreimko** [comment]
@xrn @randymetamask Hello, any updates here?

**Comment by xrn** [comment]
hi @andreimko @khkwan0 YLD symbol is already used in .json file. At this stage, MM does not have proper handling for such cases. I will merge this PR when such a feature will be released. Thanks for your patience.

**Comment by andreimko** [comment]
> hi @andreimko @khkwan0 YLD symbol is already used in .json file. At this stage, MM does not have proper handling for such cases. I will merge this PR when such a feature will be released. Thanks for your patience.

Hello, is there any plan to release proper handling for such cases this year?

**Comment by andreimko** [comment]
@xrn Hello, blocking token has changed the name: https://kanpeki.medium.com/we-are-now-ahem-kanpeki-c6b163de71ba

Can we now finally get merged?

**Comment by xrn** [comment]
Please create PR for Kanpeki (KAE) token and I will merge if everything will be correct

**Comment by khkwan0** [comment]
i updated the pull request to include the ticker name change to KAE for Kanpeki

**Comment by andreimko** [comment]
> Please create PR for Kanpeki (KAE) token and I will merge if everything will be correct

@xrn Hopefully we are good now.

**Comment by xrn** [comment]
@khkwan0 @andreimko still not - this contract is YLD this is what is written in the contract. This PR as of now has to stay in its current format as long as MM is not supporting overlapping symnols. Would be awesome if you can revert the symbol.

Please add new PR with a new contract 0x65Def5029A0e7591e46B38742bFEdd1Fb7b24436 and new contract detials/logo. After review I will merge it

**Comment by khkwan0** [comment]
Updated contract-map.json, changed contract for KAE to 0x65Def5029A0e7591e46B38742bFEdd1Fb7b24436, added new contract details/logo.

File: contract-map.json
Comment (xrn) [inline contract-map.json:3200]: Before I will be able to merge Kanpeki please remove this one as well as yld file and open this in new PR. I can not make a merge for overlapped symbols in this repo

**Comment by khkwan0** [comment]
i have removed what you have requested.  i will make a new pr as soon as this is merged and closed.

**Comment by xrn** [comment]
@khkwan0 we are almost there. Can you please do not remove 0xDcB01cc464238396E213a6fDd933E36796eAfF9f but only add 0x65Def5029A0e7591e46B38742bFEdd1Fb7b24436 details?

**Comment by khkwan0** [comment]
sorry, i'm a little confused now.  earlier you wanted me to remove 0xF9, which is yield app.  and i also removed the file.  as requested.  I did not remove 0xDcB01cc464238396E213a6fDd933E36796eAfF9f.

**Comment by khkwan0** [comment]
would it better if we just closed this pr without merging and i start new pr from a clone of a newer master?

**Comment by khkwan0** [comment]
If I add back 0xDcB01cc464238396E213a6fDd933E36796eAfF9f and it's details, then we will have the same problem again no?  Two contracts with the same ticker?

**Comment by xrn** [comment]
No, it is fine but please check in this PR files changed - https://github.com/MetaMask/contract-metadata/pull/903/files - you are removing few lines which you should not

**Comment by khkwan0** [comment]
Ok, I have readded 0xDC

**Review by xrn — APPROVED** [review]

File: contract-map.json
Comment (danfinlay) [inline contract-map.json:2890]: You have an extra space at the end of the key here.

**Comment by khkwan0** [comment]
@danfinlay this has been fixed.  Thanks

**Comment by andreimko** [comment]
@xrn Hello, any timeline when MetaMask will release this change?

**Pull request merged**
